doc/user/content/ingest-data/mysql/amazon-rds.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Ingest data from Amazon RDS for MySQL"
+description: "How to stream data from Amazon RDS for MySQL to Materialize"
+menu:
+  main:
+    parent: "mysql"
+    name: "Amazon RDS"
+    identifier: "amazon-rds-mysql"
+---
+
+This page shows you how to stream data from [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql/)
+to Materialize using the [MySQL source](/sql/create-source/mysql).
+
+## Before you begin
+
+{{% mysql-direct/before-you-begin %}}
+
+## Step 1. Enable GTID-based replication
+
+Before creating a source in Materialize, you **must** configure Amazon RDS for
+GTID-based binlog replication. This requires the following configuration
+changes:
+
+Configuration parameter          | Value  | Details
+---------------------------------|--------| -------------------------------
+`binlog_format`                  | `ROW`  | This configuration is [deprecated as of MySQL 8.0.34](https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_format). Newer versions of MySQL default to row-based logging.
+`binlog_row_image`               | `FULL` |
+`gtid_mode`                      | `ON`   | In the AWS console, this parameter appears as `gtid-mode`.
+`enforce_gtid_consistency`       | `ON`   |
+`binlog retention hours`         | 168    |
```

Comment:
I think this RDS guide is missing the step/note about the stored procedure in the database that controls binlog replication

Reply:
I just added the `log_bin` variable to the list, which was missing. In the PostgreSQL guides, we point to the official docs for most services, so this is an okay slate to start from. I'll open an issue to improve this guidance as a follow-up.